roles/config_ara_recording/tasks/main.yml

```diff
@@ -0,0 +1,76 @@
+---
+- name: Get correct python path
+  set_fact:
+    python_bin_path: "{{ ansible_facts.python.executable | dirname }}"
+  when: python_bin_path is not defined or python_bin_path == ''
+
+- name: Install ARA
+  pip:
+    name: "ara"
+    executable: "{{ (python_bin_path, 'pip') | path_join }}"
+
+- name: Ensure [ara] section in ansible.cfg exists
+  ansible.builtin.lineinfile:
+    path: "{{ config_to_edit }}"
+    regexp: '^\[ara\]'
+    line: "\n[ara]"
+    state: present
+
+- name: Add ara to callback_whitelist
+  ansible.builtin.lineinfile:
+    path: "{{ config_to_edit }}"
+    regexp: '^callback_whitelist'
+    line: 'callback_whitelist = ara'
+    state: present
+
+- name: Set ARA api_client to http
+  ansible.builtin.lineinfile:
+    path: "{{ config_to_edit }}"
+    regexp: '^api_client'
+    insertafter: '^\[ara\]'
+    line: 'api_client = http'
+    state: present
+
+- name: Set ARA api_server to the correct URL
+  ansible.builtin.lineinfile:
+    path: "{{ config_to_edit }}"
+    regexp: '^api_server'
+    insertafter: '^\[ara\]'
+    line: "api_server = http://{{ ara_server_ip }}:{{ ara_server_port }}"
+    state: present
+
+- name: Get ARA callback plugins definition
+  ansible.builtin.shell: |
+    . {{ (python_bin_path, 'activate') | path_join }}
```

Comment:
This works only in a virtual environment, not with natively installed python i.e. `/usr/bin/python`
```
TASK [config_ara_recording : Get ARA callback plugins definition] 
fatal: [localhost]: FAILED! => changed=true 
  cmd: |-
    . /usr/bin/activate
    python3 -m ara.setup.callback_plugins
  delta: '0:00:00.002860'
  end: '2024-05-29 13:22:11.322442'
  msg: non-zero return code
  rc: 127
  start: '2024-05-29 13:22:11.319582'
  stderr: '/bin/sh: 1: .: Can''t open /usr/bin/activate'
  stderr_lines: <omitted>
  stdout: ''
  stdout_lines: <omitted>

```

Reply:
Removed the dependency on virtual environment activation.